### DISCUSSION TITLE
.NET 5 Isolated Az Function

### DISCUSSION THREAD

**Discussion by junalmeida** [Azure/azure-functions-durable-extension#1993] [Q&A] [answered]
Any roadmap available for when to expect durable functions to debut on .NET 5 Isolated Az Func?

**Comment by cgillum** [comment, marked as answer]
The official roadmap is here: https://techcommunity.microsoft.com/t5/apps-on-azure/net-on-azure-functions-roadmap/ba-p/2197916, and calls out Durable Functions being available in the isolated .NET worker starting with our .NET 7 support:

> .NET 7 is scheduled for the second half of 2022, and we plan to support .NET 7 on day 1 exclusively in the isolated model. We expect that you’ll be able to migrate and run almost any .NET serverless workload in the isolated worker—including Durable Functions and other targeted improvements to the isolated model.

_Unofficially_, I can tell you that this work has already started and that we have a basic prototype working. However, we likely won't make it available for .NET 5 since .NET 5 will be out of support very soon. More likely you'll be able to start using it in Functions v4 with the .NET 6 isolated worker process, and then for sure when we launch the .NET 7 worker for Azure Functions.

**Reply by junalmeida** [reply]
That's great news, thank you!
